src/s2/util/bitmap/bitmap.h

```diff
@@ -41,8 +38,7 @@
 #include "absl/hash/hash.h"
 #include "s2/util/bits/bits.h"
 
-namespace util {
-namespace bitmap {
+namespace util::bitmap {
```

Comment:
If I remember correctly nested namespaces is C++17.
https://en.cppreference.com/w/cpp/compiler_support/17
It's ok for me, but if someone still use C++14 this header probably doesn't work

Reply:
@jmr

Reply:
Fixed.